File: solutions/go/etl/1/etl.go
package etl

import (
    "strings"
    "slices"
)

func Transform(in map[int][]string) map[string]int {
    result := make(map[string]int, 26)
    runeString := ""
    for r := 'a'; r <= 'z'; r++ {
        runeString = string(r)
        for points, letters := range in {
            if slices.Contains(letters, strings.ToUpper(runeString)) {
                result[runeString] = points
                break
            }
        }
    }
    return result
}

File: solutions/csharp/etl/1/Etl.cs
public static class Etl
{
    public static Dictionary<string, int> Transform(Dictionary<int, string[]> old)
    {
        var result = new Dictionary<string, int>();
        foreach (var kvp in old)
        {
            foreach (var letter in kvp.Value)
            {
                result[letter.ToLower()] = kvp.Key;
            }
        }
        return result;
    }
}